Controllers/loginController.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+
+const login = async (req, res) => {
+  try {
+    let userData = JSON.parse(fs.readFileSync("./data/user.json"));
```

Comment:
```suggestion
    const userData = JSON.parse(fs.readFileSync("./data/user.json"));
```
It's not redeclared again, so we can use `const`

Reply:
change it.
555030a5724922608f6b646c8dd7e1021724588a